pages/playlist/curated/[id].vue

```diff
@@ -8,6 +10,18 @@ const playlistId = Number(id);
 const { data: playlist } = usePlaylist({ id: playlistId });
 const { data: tracks, pending } = usePlaylistTracks({ id: playlistId });
 
+const { setCurrentTrack, addTrackToQueue } = inject(MediaPlaylistInjectionKey)!;
+
+function shuffle() {
+  // shuffle tracks and add them to the queue
+  const shuffledTracks = tracks.value?.filter((track) => true);
+  if (shuffledTracks && shuffledTracks.length > 0) {
+    shuffledTracks?.sort(() => Math.random() - 0.5);
+    setCurrentTrack(shuffledTracks.pop()!);
+    shuffledTracks.forEach((track) => addTrackToQueue(track));
```

Comment:
I think we need a way to add multiple tracks to the queue at once.
Something like:
playTracks(Track currentTrack, List<Track> queue)

Reply:
But I think this PR did what it wanted to achieve and further steps can be done in a separate PR.